buildpacks/README.md

```diff
@@ -56,30 +62,55 @@ metadata:
 spec:
   taskRef:
     name: buildpacks-v3
-  podTemplate:
-    volumes:
-    - name: my-cache
-      persistentVolumeClaim:
-        claimName: my-volume-claim
+# Uncomment the lines below to use an existing cache
+#  podTemplate:
+#    volumes:
+#    - name: my-cache
+#      persistentVolumeClaim:
+#        claimName: task-pv-claim
   inputs:
     resources:
     - name: source
       resourceSpec:
         type: git
         params:
         - name: url
-          value: https://github.com/my-user/my-repo
+          value: <your git repo, e.g. "https://github.com/buildpacks/samples">
     params:
+    - name: SOURCE_SUBPATH
+      value: <optional subpath within your source repo, e.g. "apps/java-maven">
     - name: BUILDER_IMAGE
-      value: gcr.io/cncf-buildpacks-ci/tekton-cnb-test:bionic
-    - name: CACHE
-      value: my-cache
+      value: <your builder image tag, see below for suggestions, e.g. "builder-repo/builder-image:builder-tag">
+# Uncomment the lines below to use an existing cache
+#    - name: CACHE
+#      value: my-cache
   outputs:
     resources:
     - name: image
       resourceSpec:
         type: image
         params:
         - name: url
-          value: gcr.io/my-repo/my-image
+          value: <your output image tag, e.g. "gcr.io/app-repo/app-image:app-tag">
+```
+
+### Example builders
+
+Cloud Foundry:
+ - `cloudfoundry/cnb:bionic`
+ - `cloudfoundry/cnb:cflinuxfs3`
```

Comment:
Any reason why `tiny` is missing here? Also, is it possible to link them to any resource applicable such as the docker hub page itself: 

- https://hub.docker.com/r/cloudfoundry/cnb
- https://hub.docker.com/r/heroku/buildpacks/

Reply:
Originally we had just the output of `pack suggest-builders` which includes `tiny` - looks like that was changed to be a more streamlined output, but not sure why.

Reply:
To play devil's advocate, maybe we should just list a one CF builder and one Heroku builder here, perhaps`cloudfoundry/cnb:cflinuxfs3`, since it includes all our buildpacks and potentially decreases the number of support issues we'll see of people just not having their apps detected because the accidentally used the wrong builder. Since tekton may be a less common and harder to debug use-case for lifecycle, we might want to minimize our support load. Thoughts?

Reply:
But we should definitely also link to those builder pages, though I still don't see a good explanation on which builder to use for first-time users.

Reply:
This was meant to give a list of a few builders users can get started with, but not necessarily show exactly the same output as `suggest-builders` (otherwise, we should just direct them to run that command).

Reply:
+1 on linking to a resource though

Reply:
> To play devil's advocate, maybe we should just list a one CF builder and one Heroku builder here, perhaps`cloudfoundry/cnb:cflinuxfs3`, since it includes all our buildpacks and potentially decreases the number of support issues we'll see of people just not having their apps detected because the accidentally used the wrong builder (since there's nothing describing which to pick). Since Tekton may be a less common and harder to debug use-case for lifecycle, we might want to minimize our support load. Thoughts?

I like that idea. We just need something that users can play with. Perhaps we can direct them to pack or the docs for more details on builders